### DISCUSSION TITLE
Help! Plugin CMI

### DISCUSSION THREAD

**Discussion by spikebam** [Zrips/CMI#7167] [Q&A]
Hello! How can I make sure that player messages from the local chat are not visible in the discord channel? I use the CMI plugin for the chat. P.S. I tried to add "!" to DiscordChatChannelPrefixRequiredToProcessMessage, but it doesn't help when I write !message, then the message is not duplicated from minecraft to discord, but when I write !!message, then a message with ! is duplicated from minecraft to discord, for example !message. P.S.S. I use the DiscordSRV plugin to write to the minecraft chat from discord and vice versa

**Comment by danyadanya2005** [comment]
I confirm. this is true. when is the fix?
